docs/configuration/pragmas.md

```diff
@@ -356,6 +356,24 @@ The output of this mode shows how long it takes to apply certain optimizers on t
 SET profiling_mode = 'detailed';
 ```
 
+```sql
+SET profiling_mode = 'standard';
+```
+
+#### Custom Profiling Metrics
+
+By default, all metrics are enabled, but they can be toggled on or off individually. This `PRAGMA` accepts a JSON object with the metric names as keys and a boolean value to enable or disable the metric. The metrics set by this `PRAGMA` will override the default settings.
```

Comment:
Hmm, maybe `by default, most metrics are enabled`, as we do not include the detailed profiling node metrics.

Reply:
Maybe `by default, all metrics are enabled, excluding those enabled by detailed profiling` or too long?